platform/ios/uitest/ios-tests.xcodeproj/project.pbxproj

```diff
@@ -284,6 +284,9 @@
 				LastUpgradeCheck = 0700;
 				ORGANIZATIONNAME = Mapbox;
 				TargetAttributes = {
+					DD043322196DB9BC00E6F39D = {
+						DevelopmentTeam = GJZR2MEM28;
+					};
```

Comment:
You need to select a code sign identity to run on device, and then it gets saved to the `pbxproj` like this. I'd recommend either `git add -p` avoiding it (what I do) or a commit script that removes it manually somehow maybe?